### DISCUSSION TITLE
Drag sorting with handlers hides the rows on drag

### DISCUSSION THREAD

**Discussion by drewgallagher** [ant-design/ant-design#43395] [Q&A]
After implementing the drag sorting feature documented [here](https://4x.ant.design/components/table/#components-table-demo-drag-sorting-handler), on drag the rows are being hidden for some reason. 

Attached is the code for the implementation and a video demoing the issue. 

Demo video
https://www.loom.com/share/57e809f725c64e03b839f8ab8661d3d2?sid=e98a4251-132c-4dd9-87c2-9206922b07ae

[files for drag and drop sorting.zip](https://github.com/ant-design/ant-design/files/11962089/files.for.drag.and.drop.sorting.zip)

**Comment by linxianxi** [comment]
can you try v5 drag demo? It used `dnd-kit`

**Reply by drewgallagher** [reply]
By upgrading to ant design v5, we will be introducing lots of refactoring work into our enterprise app that is not desirable. 

however, there is a v4 dnd-kit implementation I can try. 

Curious, why would those css rules even be getting set? 

It doesn't make sense to me at the moment.

**Reply by linxianxi** [reply]
Just use `dnd-kit`, no need to upgrade to v5

**Reply by drewgallagher** [reply]
Ok, is dnd-kit better?

Seems like the implementation requires more code

**Reply by linxianxi** [reply]
yes, `dnd-kit` is  better

**Reply by drewgallagher** [reply]
@linxianxi I followed your advice but I'm still running into compiler issues outlined in this video demo https://www.loom.com/share/949ec001e69e41efa24eb9570b881748?sid=37d9694f-30bf-49c7-9dcf-3c34906c86b9

Even downgrading to antd 4.24.12 does not resolve compiler issues.

Getting issues like the following:
![Screen Shot 2023-07-06 at 9 24 13 AM](https://github.com/ant-design/ant-design/assets/131900008/25268914-f1bf-4cca-9d4d-4be8c2be0499)
![Screen Shot 2023-07-06 at 9 24 04 AM](https://github.com/ant-design/ant-design/assets/131900008/d9423346-4c9c-433b-8ff8-681a84cfb44b)

**Reply by drewgallagher** [reply]
Ok fixed the compiler issues by updating my package.json to have: 
    "react-dnd": "latest",
    "react-dnd-html5-backend": "latest",
    "immutability-helper": "latest",

and running yarn
